src/constants/abacus.ts

```diff
@@ -370,3 +370,11 @@ export type PerpetualMarketOrderbookLevel = OrderbookLine & {
   mine: number | undefined;
   key: string;
 };
+
+export enum AbacusInputTypes {
```

Comment:
dang. any way to export this from abacus?

Reply:
Its just hard coded as strings in abacus atm 😢 , it def would be nice to have it be exported. Made a tech debt ticket for it https://linear.app/dydx/issue/OTE-572/[tech-debt]-export-input-types-as-enum-from-abacus